### PR TITLE
BZ1878949 Changed default info.

### PR DESCRIPTION
**Pull request by jonquilwilliams**
https://bugzilla.redhat.com/show_bug.cgi?id=1878949

QE and peer review required. Thank you.

### CODE REVIEW

File: modules/nodes-nodes-resources-configuring-about.adoc [hunk 1]
```diff
@@ -16,10 +16,10 @@ CPU and memory resources reserved for node components in {product-title} are bas
 | Resources reserved for node components. Default is none.
 
 |`system-reserved`
-| Resources reserved for the remaining system components. Default is none.
+| Resources reserved for the remaining system components. Default settings depend on the OpenShift and Machine Config Operator versions. Confirm the default `systemReserved` parameter on the `machine-config-operator` repo.
```
Comment (ahardin-rh) [inline modules/nodes-nodes-resources-configuring-about.adoc:19]: s/OpenShift/{product-title}
s/repo/repository

**Comment by ahardin-rh** [comment]
@jonquilwilliams Just a few minor suggestions from me. Overall, looks good!

**Comment by jonquilwilliams** [comment]
Hi, Ashley. I made your changes. Pls. let me know if anything further is needed. Still need to wait for QE review before merging. Thanks.

**Comment by ahardin-rh** [comment]
@jonquilwilliams The changes look great to me! Before we merge, we'll just have to squash the two commits down into one and re-push. I can show you how to do that if you need a hand. :smile:

**Comment by jonquilwilliams** [comment]
@ahardin-rh QA her verified this. Please merge and CP when you have a moment. Thanks!

**Pull request merged**